### ISSUE TITLE
Are there plans to support Oracle?

### ISSUE THREAD

**Issue by zoujiaqing** [zussel/matador#131]

**Comment by zussel** [comment]
Yes there are plans to provide oracle database support. But it won't be one of the next two release, because I'm improving http server/client module.
